context-monitoring/src/main/java/de/atb/context/monitoring/monitors/file/FileSystemMonitor.java

```diff
@@ -81,63 +54,6 @@ public FileSystemMonitor(final DataSource dataSource,
 
         this.logger.info("Initializing " + this.getClass().getSimpleName()
```

Comment:
The logging statement should be moved to `AbstractFileSystemMonitor`'s constructor.

Reply:
[x] done